### PR TITLE
Document constraints batch 6

### PR DESCRIPTION
**Pull request by Azdaroth**
@adamgrad When reviewing this  PR please be especially careful about Reviews endpoint as the scope of the changes are huge and I took some stuff from here: https://github.com/BookingSync/bookingsync-api-docs/pull/350/files

### CODE REVIEW

File: content/reference/endpoints/reviews.md [hunk 1]
```diff
@@ -3,44 +3,134 @@
 1. TOC
 {:toc}
 
+## Overview
+
+This resource represents guests (travelers)' Reviews for the host for a given [Booking](/reference/endpoints/bookings/).
```
Comment (adamgrad) [inline content/reference/endpoints/reviews.md:8]: do we need the `'` after `(travelers)`?

File: content/reference/endpoints/reviews.md [hunk 2]
```diff
@@ -3,44 +3,134 @@
...
+          |                     | rental                | Integer | Read       | Rental's ID related to the Review
+          |                     | source                | Integer | Read       | Source's ID related to the Review
+          |                     | reply                 | Integer | Read       | Reply's (ReviewReply)'s ID related to the Review
+          |                     | booking               | Integer | Read       | Booking's ID related to the Review
+ ---------|---------------------|-----------------------|---------|------------|------------
+          |                     | id                    | Integer | Read       | Review's ID. 
+          |                     | title                 | Text    | Read | Title/Summary of the review. 
+          |                     | comment               | Text    | Read | Comment from traveler (guest). 
+          |                     | private_comment       | Text    | Read | Private comment from the traveler (guest) (not meant to be exposed publicly). Requires `reviews_read` or `reviews_write` scope
```
Comment (adamgrad) [inline content/reference/endpoints/reviews.md:31]: >Requires `reviews_read` or `reviews_write` scope

do we need this comment? It usually is just reflected by displaying or not displaying an attribute in the selected oauth scope table

File: content/reference/endpoints/reviews.md [hunk 3]
```diff
@@ -3,44 +3,134 @@
...
+          |                     | rental                | Integer | Read       | Rental's ID related to the Review
+          |                     | source                | Integer | Read       | Source's ID related to the Review
+          |                     | reply                 | Integer | Read       | Reply's (ReviewReply)'s ID related to the Review
+          |                     | booking               | Integer | Read       | Booking's ID related to the Review
+ ---------|---------------------|-----------------------|---------|------------|------------
+          |                     | id                    | Integer | Read       | Review's ID. 
+          |                     | title                 | Text    | Read | Title/Summary of the review. 
+          |                     | comment               | Text    | Read | Comment from traveler (guest). 
+          |                     | private_comment       | Text    | Read | Private comment from the traveler (guest) (not meant to be exposed publicly). Requires `reviews_read` or `reviews_write` scope 
+          |                     | rating                | Integer | Read/Write | Overall rating of a stay
```
Comment (adamgrad) [inline content/reference/endpoints/reviews.md:32]: as far as I recall this was changed to a decimal already:
`required(:rating).filled(:decimal, gteq?: 1, lteq?: 5)`

File: content/reference/endpoints/reviews.md [hunk 4]
```diff
@@ -3,44 +3,134 @@
...
+          |                     | reply                 | Integer | Read       | Reply's (ReviewReply)'s ID related to the Review
+          |                     | booking               | Integer | Read       | Booking's ID related to the Review
+ ---------|---------------------|-----------------------|---------|------------|------------
+          |                     | id                    | Integer | Read       | Review's ID. 
+          |                     | title                 | Text    | Read | Title/Summary of the review. 
+          |                     | comment               | Text    | Read | Comment from traveler (guest). 
+          |                     | private_comment       | Text    | Read | Private comment from the traveler (guest) (not meant to be exposed publicly). Requires `reviews_read` or `reviews_write` scope 
+          |                     | rating                | Integer | Read/Write | Overall rating of a stay 
+          |                     | rating_accurate       | [Decimal](/reference/enums#formats)    | Read       | Non-rounded rating.
```
Comment (adamgrad) [inline content/reference/endpoints/reviews.md:33]: this is deprecated and dropped

**Comment by adamgrad** [comment]
@Azdaroth did you check if the reviews page is rendering correctly? In the GH changeset It looks a bit off in some places,  but it may be just GH display, so good to make sure it's looking good

File: content/reference/endpoints/reviews.md (hunk 3 above)
Comment (StoneFrog) [reply content/reference/endpoints/reviews.md:32]: Are you sure? 
The idea was the have rolling migration, as far as I recall, we accept decimal on write, but for read this value is passed directly as `rating_accurate` and casted to integer for `rating` (for backwards compatibility)

Comment (Azdaroth) [reply content/reference/endpoints/reviews.md:32]: Yeah, exactly, it's a decimal on write and integer on read and there is a rating accurate that is a decimal

File: content/reference/endpoints/reviews.md (hunk 4 above)
Comment (Azdaroth) [reply content/reference/endpoints/reviews.md:33]: @adamgrad how come if `rating` is an integer? what's the way to get an accurate rating then?

**Comment by Azdaroth** [comment]
@adamgrad yeah, it's ok

File: content/reference/endpoints/reviews.md (hunk 2 above)
Comment (Azdaroth) [reply content/reference/endpoints/reviews.md:31]: good point

File: content/reference/endpoints/reviews.md (hunk 1 above)
Comment (Azdaroth) [reply content/reference/endpoints/reviews.md:8]: yeah, we do. I mean in singular you have `traveler's` and in plural `travelers'`, but it should be inside parentheses and applied to `guests'` as well :P

**Comment by Azdaroth** [comment]
@adamgrad feedback applied

File: content/reference/endpoints/reviews.md (hunk 3 above)
Comment (adamgrad) [reply content/reference/endpoints/reviews.md]: hmm, didn't know that, thanks for clarification and heads up Mat 👍

**Review by adamgrad — APPROVED** [review]

**Pull request merged**